docker-vertica/packages/cleanup.sh

```diff
@@ -34,11 +34,17 @@ rm -r -f \
    /opt/vertica/examples \
    /opt/vertica/sdk \
    /opt/vertica/packages/*/examples \
-   /opt/vertica/oss/python3/lib/python3.7/test \
-   /opt/vertica/oss/python3/lib/python3.7/pip \
-   /opt/vertica/oss/python3/lib/python3.7/site-packages/pip \
-   /opt/vertica/oss/python3/lib/python3.7/config-3.7*
+   /opt/vertica/packages/kafka \
+   /opt/vertica/oss/python3/lib/python3.9/test \
```

Comment:
General: can we use wildcards for the python versions?  Will this work?
/opt/vertica/oss/python*/lib/python*/test